Keyboard accesibility for user settings dialog (#2566)
SAP/luigi: docs/user-settings.md

#### icon

- **type** string (optional)

> - **description** name of the icon, without the `sap-icon--` prefix or path to an image. It is also possible to use a picture instead of an icon via passing the right path to it. 

Reviewer: ```suggestion
- **description** name of the icon, without the `sap-icon--` prefix. It is also possible to use an image instead of an icon by specifying its path.  
```
